jenkins/test/testsuite/Jenkinsfile

```diff
@@ -60,7 +60,7 @@ pipeline {
                 script {
                     basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"
                     sh "wget ${basePath}/builds/opensearch/manifest.yml"
-                    sh "./test.sh ${JOB_NAME} ${basePath} --test-run-id ${env.BUILD_NUMBER}"
+                    sh "./test.sh ${JOB_NAME} manifests/${opensearch_version}/opensearch-${opensearch_version}-test.yml ${basePath} --test-run-id ${env.BUILD_NUMBER}"
                 }
             }
             post() {
```

Comment:
For later: we should probably move this file under `opensearch/` since we'll need one of these for Dashboards.

Reply:
Dashboards will have a test manifest with "opensearch-dashboards". Thus sub dir is no needed due to different names.